vpr/src/route/rr_graph2.cpp

```diff
@@ -2486,7 +2306,7 @@ static void label_wire_muxes(const int chan_num,
      * If seg_type_index == UNDEFINED, all segments in the channel are considered. Otherwise this routine
      * only looks at segments that belong to the specified segment type. */
 
-    std::vector<int>& labels = *labels_ptr;
```

Comment:
Are these "labels" actually RRNodeIDs, or are they track numbers, or something else?  Depending on the answer, they should stay ints, or be converted to RRNodeIds as this PR does.  The comment should be updated to explain what a label is (is it an RRNodeId, a track number, or something else?